src/main/java/com/sixt/service/framework/rpc/RpcCallException.java

```diff
@@ -200,15 +201,20 @@ private static String getMessage(JsonObject object) {
         return message;
     }
 
+    /**
+     * provides a {@link Category} by the given {@link JsonObject}.
+     *
+     * @param object JSON element
+     * @return Category that is provided in JSON, if the code is not known it will fall back to {@link Category#InternalServerError}.
+     */
     private static Category getCategory(JsonObject object) {
-        // if no category can be found we use internal server error
         Category category = Category.InternalServerError;
         if (object.has(CATEGORY)) {
```

Comment:
can you please just harden this a bit since we are here?  getAsInt can throw ClassCastException or IllegalStateException